openapi/api.yml

```diff
@@ -0,0 +1,204 @@
+openapi: "3.0.0"
+
+info:
+  description: Fund store API for DLUHC Funding Service Design
+  version: "0.1.0"
+  title: Funding Service Design - Fund store.
+
+paths:
+  /funds:
+    get:
+      tags:
+      - Funds
+      summary: Returns list of all funds
+      description: Returns list of all funds
```

Comment:
🔧 Add description of searching functionality.

Reply:
👌